### ISSUE TITLE
Scheduler warning

### ISSUE THREAD

**Issue by rasbt** [Lightning-AI/litgpt#1561]
### Bug description

When finetuning models, we get this warning:

```
Epoch 1 | iter 1597 step 99 | loss train: 7.316, val: n/a | iter time: 33.55 ms
Epoch 1 | iter 1598 step 99 | loss train: 7.357, val: n/a | iter time: 33.57 ms
Epoch 1 | iter 1599 step 99 | loss train: 7.249, val: n/a | iter time: 33.63 ms
/home/zeus/miniconda3/envs/cloudspace/lib/python3.10/site-packages/torch/optim/lr_scheduler.py:156: UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible. Please use `scheduler.step()` to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available. Please open an issue if you are unable to replicate your use case: https://github.com/pytorch/pytorch/issues/new/choose.
  warnings.warn(EPOCH_DEPRECATION_WARNING, UserWarning)
Epoch 1 | iter 1600 step 100 | loss train: 7.141, val: n/a | iter time: 35.37 ms (step)
Validating ...
```

 I checked all finetuning codes, and we use `scheduler.step()` without any args. Any ideas where it could come from @Andrei-Aksionov @awaelchli ?

### What operating system are you using?

Linux

### LitGPT Version

``` 

main branch

```

**Comment by awaelchli** [comment]
It's a bug in PyTorch and needs to be addressed there (from the internals of SequentialLR). There are several such reports, for example https://github.com/pytorch/pytorch/issues/130018. If you don't want the warning to be shown, one solution can be to suppress it by adding a warning filter to litgpt.

**Comment by rasbt** [comment]
Great, thanks @awaelchli , I added a PR in #1562 to address this.
